client/crypto/crypto.go

```diff
@@ -0,0 +1,100 @@
+package crypto
+
+import (
+	"errors"
+
+	"github.com/yunify/qingstor-sdk-go/service"
+)
+
+// Algorithm is the set of all unsigned 64-bit integers.
+// Range: 0 through 18446744073709551615.
+type Algorithm uint64
+
```

Comment:
Need to support 18446744073709551615 kinds of crypto algorithm?

Are you kidding me?

Reply:
// AES256 represents aes56 algorithm
 AES256 Algorithm = 1 << iota
There are only 128 types of algorithms
OK, I modify `uint64` to `uint8`.